### DISCUSSION TITLE
Color profiles for each client

### DISCUSSION THREAD

**Discussion by MichaelIspas** [ClassicOldSong/Apollo#146] [Q&A]
Hello, I love using Apollo and am close to being done with my setup. I stream to a Steam Deck Oled and also docked to a 4k tv. I am having difficulty having specific applications remember the primary ICC profile I assigned it to. Sometimes it works sometimes it doesn’t. I have one for docked and one for handheld mode added under Applications.

My deck has a max of 1000 nits so I made a profile for that client, same with another application for when it’s docked with 1300 nits display so you get the idea. Do VDD cover that entirely or not quite? Thanks.

**Comment by ClassicOldSong** [comment]
Theoretically it should be handled automatically by Windows but apparently it's really buggy at this moment... I have no idea how to workaround this issue despite wait for Windows to fix the problem.

**Reply by MichaelIspas** [reply]
Would something like this work for the time being? I am wondering if I can utilize this somehow in Apollo settings via commands, I'll see what I can do.

https://github.com/timsutton/customdisplayprofiles

**Reply by ClassicOldSong** [reply]
This tool only works in macOS. A quick search lead me to this: https://www.argyllcms.com/

It seems that you need to install a tollset from this site and use `displaywin` to load the profiles.

**Reply by MichaelIspas** [reply]
Silly me, I missed that part about it being Mac only. That's a good source. I also found this (Windows 7/10 but could work on 11) and a couple reviews talk about sunshine and virtual displays.

https://sourceforge.net/projects/monitorswitcher/

**Reply by ClassicOldSong** [reply]
I think this one only changes resolution/refreshrate and display combinations.
